schema/schema.go

```diff
@@ -39,7 +39,7 @@ const (
 	// NoneSchemaName references a disabled/NOP schema for Load()/Set().
 	NoneSchemaName = "none"
 	// DefaultSchemaName is the none schema.
```

Comment:
This comment is out of sync now.

Reply:
Thanks for spotting. Removed it as it is not used now.